### DISCUSSION TITLE
How to reproduce a variation?

### DISCUSSION THREAD

**Discussion by AndrWeisR** [hlky/stable-diffusion-webui#1004] [Q&A]
I find a nice image with a given prompt and seed, but I used a variation of 0.25. I have a yaml file produced for the image, but I can't reproduce the image with the same prompt and seed, because I don't have exactly what the variation changed, I guess. Is there a way of getting the same image back that was produced as a variation?

**Comment by mykeehu** [comment]
I found a seed identifier in the yaml file and entered it in the Variant Seed field, but to no avail, I didn't get the same image back, even though all the other parameters were correct. In addition, the regenerated image had a different seed value.
I don't know how this seed ID is calculated.
